teps/0078-tekton-catalog-support-tiers.md

```diff
@@ -0,0 +1,345 @@
+-->
+
+As discussed in [TEP-0003: Tekton Catalog Organization](https://github.com/tektoncd/community/blob/main/teps/0003-tekton-catalog-organization.md), 
+the support tiers for resources in the Tekton Catalog are *commmunity*, *verified* and *official*, which are 
+differentiated as such:
+
+|                        | Community |      Verified      |      Official      |
+|:----------------------:|:---------:|:------------------:|:------------------:|
+|   Automated Testing    |    :x:    | :heavy_check_mark: | :heavy_check_mark: |
+| Images scanned for CVE |    :x:    |        :x:         | :heavy_check_mark: |
+|  Maintained by Tekton  |    :x:    |        :x:         | :heavy_check_mark: |
+
+Today, all the resources in the Tekton Catalog and Hub should be in the *community* support tier, however this 
+information is not surfaced to users. 
+
+The aim of this TEP is surface information about the support tiers for resources in the Tekton Catalog, Hub and CLI. 
+In addition, this TEP aims to enable users to filter and search resources from the Tekton Catalog, Hub and CLI for 
+specific support tiers. Providing this information in a user-friendly way will set us up well to support the *verified*
+and *official* resources in the Tekton Catalog, Hub and CLI.
+
+### Goals
+
+<!--
+List the specific goals of the TEP.  What is it trying to achieve?  How will we
+know that this has succeeded?
+-->
+
+- Define how to add support tiers into resource specifications in the Tekton Catalog. 
+- Define how to show the support tiers to users and how to enable users to search and filter based on support tiers.
+
+### Non-Goals
+
+<!--
+What is out of scope for this TEP?  Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+- Establish *verified* and *official* support tiers for resources in the Tekton Catalog - will be done in future work.
+- Define criteria to promote or demote the support tiers of resources.
+
+### Use Cases
+
+<!--
+Describe the concrete improvement specific groups of users will see if the
+Motivations in this doc result in a fix or feature.
+
+Consider both the user's role (are they a Task author? Catalog Task user?
+Cluster Admin? etc...) and experience (what workflows or actions are enhanced
+if this problem is solved?).
+-->
+
+- Indicate the support tier of resources in the Tekton Catalog and Hub.
+- Discover the support tier of a specific resource in the Tekton Catalog and Hub.
+- Filter resources in the Tekton Hub based on support tiers.
+- Search resources with the Tekton CLI for resources in certain support tiers.
+
+## Requirements
+
+<!--
+Describe constraints on the solution that must be met. Examples might include
+performance characteristics that must be met, specific edge cases that must
+be handled, or user scenarios that will be affected and must be accomodated.
+-->
+
+- Contributor should be able to indicate the support tier of a resource when authoring it.
```

Comment:
From what I have seen on the catalog a lot of the contribution are from contributor that contribute task once and not necessary follow up if there is a catalog structure change or an issue/rfe reported unless catalog owners ping them.

If contributor start to indicate that this task is supported, the expectation of the support needs really be on the contributor and not on the catalog owners (since that would not scale).

Reply:
definitely, the contributor needs to support their resources if they're in the `verified` tier -- so the catalog owners only maintain the `official` tier 

@vinamra28 and I have been discussing ideas to automate notifying the owners (contributors) about failing tests for their resources (without catalog owners having to ping) -- and right now, we're thinking of creating github issues with a given format and assigning the owners of the specific failing resource (more on this when we get to the proposal bit of https://github.com/tektoncd/community/pull/495, trying to keep this TEP focused on the user experience)